File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/batch_list_response.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class BatchListResponse:
    """OpenAI batch list response"""
    object: str = None
    data: List[BatchResponse] = None
    has_more: bool = None
    first_id: str = None
    last_id: str = None

File: sdkwork-ai-sdk-typescript/src/types/batch-list-response.ts
import type { BatchResponse } from './batch-response';

/** OpenAI batch list response */
export interface BatchListResponse {
  /** Object type */
  object?: string;
  /** Batch list */
  data?: BatchResponse[];
  /** Whether more data exists */
  has_more?: boolean;
  /** First item id */
  first_id?: string;
  /** Last item id */
  last_id?: string;
}
